blueprints/app/files/app/styles/app.css

```diff
@@ -0,0 +1 @@
+/* placeholder */
```

Comment:
Needed to avoid a lint violation:

```
app/styles/app.css
 1:1  ✖  Unexpected empty source  no-empty-source
```

Reply:
That's one possibility. However, I lean against that because:

* It gets messy having to include disable comments in the blueprint
* It makes this file "aware" of stylelint
* If someone starts adding actual styles to `app.css`, they have to know to remove this stylelint disable comment (I'm sure we would all know that, but can't assume everyone would know to clean it up)

Instead, what I recommend is we add a brief, helpful comment to explain what `app.css` is for, perhaps just:

```css
/* Ember CLI supports plain CSS out of the box. You can add your CSS styles to here. More info: https://cli.emberjs.com/release/advanced-use/stylesheets/ */
```

That would be useful regardless of the presence of stylelint / this violation.

Reply:
Seems much better than just `/* placeholder */` to me 👍